### PR TITLE
[ConvDiff] Mixed Laplacian element

### PR DESCRIPTION
**Pull request by rubenzorrilla**
**Description**
This PR adds a new mixed (scalar-gradient) Laplacian formulation (primal) to the `ConvectionDiffusionApplication`. It is compatible with the already existent conditions.

### CODE REVIEW

File: kratos/includes/convection_diffusion_settings.h [hunk 1]
```diff
@@ -27,6 +27,7 @@
 
 // Project includes
 #include "includes/define.h"
+#include "includes/variables.h"
```
Comment (philbucher) [inline kratos/includes/convection_diffusion_settings.h]:
```suggestion
#include "containers/variable.h"
```

Comment (philbucher) [reply kratos/includes/convection_diffusion_settings.h]: wasn't this in another PR these days?

Comment (rubenzorrilla) [reply kratos/includes/convection_diffusion_settings.h]: Yes, most probably I messed something when switching branches. I'll revert in here.

Comment (rubenzorrilla) [reply kratos/includes/convection_diffusion_settings.h]: I've just checked and it is required in here as well. Indeed, we need to include both `variable.h` and `kratos_components.h` in order to make it work.

Comment (philbucher) [reply kratos/includes/convection_diffusion_settings.h]: makes sense to me, still better than `variables.h`

Comment (rubenzorrilla) [reply kratos/includes/convection_diffusion_settings.h]: I'd cerry-pick the commit in the other PR as well.

**Review by RiccardoRossi — APPROVED** [review]

**Pull request merged**